doc_source/command-options.md

```diff
@@ -38,6 +38,8 @@ For example, both the EB CLI and Elastic Beanstalk console set the configuration
 Because the recommended values are set at the API level, they will override values for the same options that you set in configuration files or saved configurations\. The following options are set:
 
 **Elastic Beanstalk console**
++ Namespace: `aws:autoscaling:asg`
+  Option Names: `HealthCheckType`, `HealthCheckGracePeriod`
```

Comment:
this aren't valid as-is... where could I find the correct syntax?

Reply:
Hello @mathieujobin -

Here are other topics and guides that provide more information on the options and settings you mention:

- The **_[aws:autoscaling:asg](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html#command-options-general-autoscalingasg)_** namespace is documented in the [General options](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html) topic in the _Elastic Beanstalk Developer Guide_.  This namespace is not included in the [Recommended values](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options.html#configuration-options-recommendedvalues) topic, since its options are not overridden.
- The _HealthCheckType_ and _HealthCheckGracePeriod_ options belong to the _**[AWS::AutoScaling::AutoScalingGroup](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html)**_  namespace, which is part of the _EC2 Auto Scaling_ service.   The [Auto Scaling health check setting](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environmentconfig-autoscaling-healthchecktype.html) topic in the _Elastic Beanstalk Developer Guide_ mentions these options, and refers readers to the the _AWS CloudFormation User Guide_ and [Health Checks for Auto Scaling Instances](https://docs.aws.amazon.com/autoscaling/ec2/userguide/healthcheck.html) in the _Amazon EC2 Auto Scaling User Guide_ for more information.
- The _**[aws:elb:policies](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html#command-options-general-elbpolicies)**_ namespace is explained further in the [General options](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html) topic of the _Elastic Beanstalk Developer Guide_.  The [Classic Load Balancer topic](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environments-cfg-clb.html#environments-cfg-clb-namespace) of the _Configuring environments_ chapter in the _Elastic Beanstalk Developer Guide_ also mentions this option.

I hope these references are helpful.  Thank you for your post, and please accept my apologies for the delayed response.